src/replay_syscall.cc

```diff
@@ -1269,6 +1269,7 @@ static void rep_process_syscall_arch(ReplayTask* t, ReplayTraceStep* step,
         }
       }
     }
+      /* FALLTHROUGH */
```

Comment:
This is the simplest way to do it without me having to think about where to put the `ifdef`s...... So `[[fallthrough]]` for c++17 and then `__attribute__((fallthrough))` for gcc 7 ?